nullaway/src/test/java/com/uber/nullaway/jspecify/GenericMethodTests.java

```diff
@@ -820,7 +820,7 @@ public void varargsInference() {
             "    Foo<String> foo3 = make(\"hello\", null, \"world\");",
             "    Foo<@Nullable String> foo4 = make(\"hello\", \"world\");",
             "    Foo<@Nullable String> foo5 = make(\"hello\", \"world\", makeStr(null));",
-            "    // BUG: Diagnostic contains: passing @Nullable parameter 'makeStr(null)' where @NonNull is required",
+            "    // BUG: Diagnostic contains: passing @Nullable parameter 'null' where @NonNull is required",
```

Comment:
I'm not 100% sure why this error message changed.  But the new message is as valid as the old one, and given all the other tests still pass, I'm not concerned.